SoomlaiOSProfile/ProfileEventHandling.h

```diff
@@ -54,6 +56,17 @@
 #define EVENT_UP_INVITE_CANCELLED               @"up_invite_cancelled"
 #define EVENT_UP_INVITE_FAILED                  @"up_invite_failed"
 
+#define EVENT_UP_GAME_LEADERBOARDS_STARTED      @"up_game_leaderboards_started"
+#define EVENT_UP_GAME_LEADERBOARDS_FINISHED     @"up_game_leaderboards_finished"
+#define EVENT_UP_GAME_LEADERBOARDS_FAILED       @"up_game_leaderboards_failed"
+
+#define EVENT_UP_GAME_SCORES_STARTED            @"up_game_scores_started"
+#define EVENT_UP_GAME_SCORES_FINISHED           @"up_game_scores_finished"
+#define EVENT_UP_GAME_SCORES_FAILED             @"up_game_scores_failed"
+
+#define EVENT_UP_SCORE_REPORT_STARTED           @"up_game_scores_started"
```

Comment:
@EugeneButusov 
1. You have the same strings here.
2. I don't like the names too much. Why do we have GAME in there? I think it should be something like `GET_LEADERBOARDS_STARTED` `GET_SCORES_STARTED`  `REPORT_SCORES_STARTED`